tests/conventions/test_ugrid.py

```diff
@@ -114,6 +114,7 @@ def make_dataset(
     time_size: int = 4,
     make_edges: bool = True,
     make_face_coordinates: bool = False,
+    set_coordinates_as_coordinates: bool = False,
```

Comment:
Can you set this default to `True`? Given that xarray is slowly getting better at coordinate, this being True represents the likely future better, plus it might expose more cases where we've made incorrect assumptions about data_vars vs. coordinates.